src/test/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/AddressBookParserTest.java

```diff
@@ -138,6 +139,13 @@ public void parseCommand_listIngredient() throws Exception {
         assertTrue(parser.parseCommand(ListIngredientCommand.COMMAND_WORD + " 3") instanceof ListIngredientCommand);
     }
 
+    @Test
+    public void parseCommand_deleteIngredient() throws Exception {
+        DeleteIngredientCommand command = (DeleteIngredientCommand) parser.parseCommand(
+                DeleteIngredientCommand.COMMAND_WORD + " " + INDEX_FIRST.getOneBased());
+        assertEquals(new DeleteIngredientCommand(INDEX_FIRST), command);
+    }
```

Comment:
Can you move this method up and place it after `parseCommand_deleteClient()` to maintain lexicographic order